ios/RNTPTDocumentView.h

```diff
@@ -62,6 +62,7 @@ static NSString * const PTAnnotationCreateAreaMeasurementToolKey = @"AnnotationC
 static NSString * const PTPanToolKey = @"pan";
 static NSString * const PTAnnotationEraserToolKey = @"AnnotationEraserTool";
 static NSString * const PTAnnotationCreateSoundToolKey = @"AnnotationCreateSound";
+static NSString * const PTPencilDrawingToolKey = @"PencilDrawing";
```

Comment:
Let's call all these references it "Pencil" instead "PencilKit", because it's the latter, and the Apple Pencil can be used to create other types of annots too.